### DISCUSSION TITLE
Python Packages Support Timeline?

### DISCUSSION THREAD

**Discussion by tarrenj** [community/community#24793] [Apps, API and Webhooks]
<p>Is there an existing feature request for adding a Python index to the suppported clients/formats, or should I submit one?</p>
<p>I did a quick search on the forum but couldn’t find anything, appologies if this has already been asked and answered.</p>
<p>Thanks!</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey! Feature requests should go directly to GitHub support so checking the forum may not be the best way to discern if someone else has already requested it. It’s best to request the feature directly via the support contact. </p>
